src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -549,6 +552,10 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
             if (_lo.get_optimization_attributes().use_onednn_impls == 1)
                 return true;
 
+            if (node.get_output_layout().is_dynamic()) {
+                return true;
+            }
```

Comment:
shouldnt it be qpplied for other primitives? not only conv..

Reply:
It should, but this is not the goal of this PR. We'll need to check fusion passes more closely and add proper test coverage

Reply:
I see.